docker-compose.yaml

```
@@ -10,8 +10,8 @@ x-common-environment: &common-environment
   HEALTH_CHECK_MAX_RETRY_INTERVAL_SECONDS: 10
   HEALTH_CHECK_SUCCESS_THRESHOLD: 10
   CAPACITY_LIMIT: '{"type":"percentage", "value":80}'
-  SIWF_URL: 'https://projectlibertylabs.github.io/siwf/ui'
-  SIWF_DOMAIN: 'localhost'
+  SIWF_URL: "https://projectlibertylabs.github.io/siwf/v1/ui"
+  SIWF_DOMAIN: "localhost"
```

nit: this file uses `'` for all other strings
```suggestion
  SIWF_URL: 'https://projectlibertylabs.github.io/siwf/v1/ui'
  SIWF_DOMAIN: 'localhost'
```